user-interface/src/lib/components/Header.scss

```diff
@@ -240,3 +238,33 @@
     }
   }
 }
+
+@media screen and (max-width: 74em) {
+  .cams-header.usa-header--basic {
+    .usa-nav-container {
+      .cams-logo-and-title {
+        .site-title.wide-screen {
+          display: none;
+        }
+        .site-title.small-screen {
+          display: flex;
+        }
+      }
+    }
+  }
+}
+
+@media (min-width: 73em) {
+  .cams-header.usa-header--basic {
+    .usa-nav-container {
+      .cams-logo-and-title {
+        .site-title.wide-screen {
+          display: flex;
+        }
+        .site-title.small-screen {
+          display: none;
+        }
+      }
+    }
+  }
+}
```

Comment:
Do these widths not have an overlap? It seems like from 73 to 74 em we would have ambiguity as to which style applies.

Reply:
Great point.